lib/file-helper.js

```diff
@@ -0,0 +1,10 @@
+'use strict'
+
+const fs = require('fs')
+
+// This helper function takes a path of a nunjucks file and
+// returns the first macro code block inside that file
+exports.getRawMacro = path => {
+  let data = fs.readFileSync(path)
+  return data.toString().split('{% macro code() %}')[1].split('{% endmacro %}')[0]
```

Comment:
This is a wider question and not necessarily within the bounds of this PR but should we have some error checking here to throw up a friendly error message in case the file doesn't contain what you expect? Or do we expect any issues with Nunjucks files to have been picked up in previous reviews? Thoughts?

Reply:
Split the function and catched errors on both file reading and file processing.